lib/typescript/render/providers/chatplugin/ChatPluginRender.tsx

```diff
@@ -23,19 +38,54 @@ function render(content: ContentUnion, props: MessageRenderProps) {
           suggestions={content.suggestions}
         />
       );
+
+    // TODO render more chatplugin models
   }
 }
 
-function mapContent(message: Message): ContentUnion {
+// TODO map more string content to chatplugin models
+function chatpluginInbound(message: Message): ContentUnion {
```

Comment:
I think for chatplugin in-/ outbound are the same so you can have one method instead